Src/PerformanceCollector/Shared/PerformanceCollectorModule.cs

```diff
@@ -125,6 +131,12 @@ public void Initialize(TelemetryConfiguration configuration)
                             throw new ArgumentNullException("configuration");
                         }
 
+                        if (!this.EnableIISExpressPerformanceCounters && IsRunningUnderIisExpress())
+                        {
+                            PerformanceCollectorEventSource.Log.RunningUnderIisExpress();
```

Comment:
I think @SoubhagyaDash meant to notify customer in debug output. Like `Debug.Write("Counters disabled on IISexpress")`. I'm fine with Verbose